src/pages/posts/postsList.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="page-index">
+    <q-page class="flex flex-center">
+      <div v-for="value in TesteContent" :key="value">
+        <card-post :content="value"/>
+      </div>
+    </q-page>
+    <div class=" flex flex-center">
+      <q-pagination v-model="current" :max="5" />
+    </div>
+  </div>
+</template>
+
+<script>
+import CardPost from 'src/components/CardPost.vue'
+
+export default {
+  data () {
+    return {
+      current: 1
+    }
+  },
+  components: {
+    CardPost
+  },
+
+  computed: {
+    TesteContent () {
```

Comment:
utilizar camelCase, e o nome poderia ser `postsList` não?

Reply:
não faria sentido utilizar um nome para o que a seção faz? se ela listar post é valido postsList por exemplo, se for outra função o nome deve dizer o que aquela seção é ou faz no código.